feeds/ucentral/poe/files/05_poe

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Hi Jack,
please convert this file to use the uci-defaults feature
here is an example
--> https://github.com/Telecominfraproject/wlan-ap/blob/main/feeds/ucentral/ucentral-schema/files/etc/uci-defaults/99-ucentral-event
uci-default files get executed once after firstboot/factory-reset and then deleted
you can modify /etc/config/poe directly without worrying about /etc/config-shadow/
once all uci-default scripts were executed, the system will automatically generate the /etc/config-shadow/ folder
this should make this script much smaller and simpler